__tests__/app.test.js

```diff
@@ -143,6 +143,8 @@ describe("/api/articles/:article_id/comments", () => {
         const { comments } = res.body;
         //if topic is an array
         expect(comments).toBeInstanceOf(Array);
+        //comments should be served with the most recent comments first
```

Comment:
Extra test needed for comments to be an empty array for valid review_id with no comments